thirdparty/libktx/LICENSE.adoc

```diff
@@ -0,0 +1,10 @@
+= LICENSE file for the KhronosGroup/dfdutils project
```

Comment:
With two license files it's unclear which is which, this one should be renamed to `LICENSE.dfdutils.adoc`.